packages/use-context-menu/package.json

```diff
@@ -21,7 +21,8 @@
       "import": "./dist/use-context-menu.cjs.mjs",
       "default": "./dist/use-context-menu.cjs.js"
     },
-    "./package.json": "./package.json"
+    "./package.json": "./package.json",
+    "./styles.css": "./styles.css"
```

Comment:
The idea behind `package.json#exports` is that it should define everything that is exposed by the package so I think that this should be explicitly listed here. I'm not sure how strict bundlers are about this when it comes to non-JS stuff but better to be safe than sorry.

Reply:
> I checked this with npm pack to verify that the file was being included in the bundle, but fair enough.

It's not about what gets published. You could have hundreds of files internally but just expose 1 for public consumption. So its's more about the import site in users' projects:
```ts
import 'use-context-menu/styles.css'
```

Notice that we already expose `package.json` automatically as that's something that some tools often want to `require`. There is no big harm in allowing that but without this explicit entry `require('use-context-menu/package.json')` would throw.